q2_fmt/plugin_setup.py

```diff
@@ -299,5 +300,48 @@
     }
 )
 
+plugin.methods.register_function(
+    function=q2_fmt.peds_bootstrap,
+    inputs={'table': FeatureTable[Frequency | RelativeFrequency |
+                                  PresenceAbsence]},
+    parameters={'metadata': Metadata, 'time_column': Str,
+                'reference_column': Str, 'subject_column': T_subject,
+                'filter_missing_references': Bool,
+                'drop_incomplete_subjects': Bool,
+                'drop_incomplete_timepoint': List[Str],
+                'bootstrap_replicates': Int % Range(999, None)},
+    outputs=[('stats', StatsTable[Pairwise])],
+    parameter_descriptions={
+        'metadata': 'The sample metadata.',
+        'time_column': 'The column within the `metadata` that the'
+                       ' `table` should be grouped by. This column'
+                       ' should contain simple integer values.',
+        'reference_column': 'The column within the `metadata` that contains'
+                            ' the sample to use as a reference'
+                            ' for a given `table`.'
+                            ' For example, this may be the relevant donor'
+                            ' sample to compare against.',
+        'subject_column': 'The column within the `metadata` that contains the'
+                          ' subject ID to be tracked against timepoints.',
+        'filter_missing_references': 'Filter out references contained within'
+                                     ' the metadata that are not present'
+                                     ' in the table.'
+                                     ' Default behavior is to raise an error.',
+        'drop_incomplete_subjects': 'Filter out subjects that do not have'
+                                    ' a sample at every timepoint.'
+                                    ' Default behavior is to raise an error.',
+    },
+    output_descriptions={
+        'stats': 'stats table for comparing real PEDS values'
+        ' from fake Bootstrapped values'
```

Comment:
I'm not sure how to interpret this - will try to come up with a suggestion after I review the function, but just a note that we should improve this help text.